contribution/translation/prevalidator.php

```diff
@@ -263,31 +315,85 @@ private function check_missing_keys($reference_file, $uploaded_file)
 		return $missing_keys;
 	}
 
+	/**
+	 * Add license file
+	 * @param $license_file
+	 * @param $reference_path
+	 */
 	private function add_license_files($license_file, $reference_path)
 	{
 		$res = fopen($license_file, 'w');
-		fwrite($res, file_get_contents($reference_path . '/docs/COPYING'));
+		fwrite($res, file_get_contents($reference_path . 'docs/LICENSE.txt'));
```

Comment:
I don't think we use the .txt extension?

Reply:
I assume it's this file? https://github.com/phpbb/customisation-db/blob/3.2.x/docs/LICENSE.txt

Reply:
It's the phpBB one: https://github.com/phpbb/phpbb/blob/master/phpBB/docs/LICENSE.txt

The package gets extracted within Titania, so line 326 above would resolve to a path like:

    ./ext/phpbb/titania/store/extracted/3.2.3/docs/LICENSE.txt